reference_plate_data/README.md

```diff
@@ -0,0 +1,23 @@
+# Barcode Platemap and Platemaps
+
+In this folder, there three file types:
+
+1. **Barcode plate** - CSV file that associates each plate to the correct platemap
+2. **Platemap** - TXT that contains the plate layout (e.g., what broad sample is in what well)
+3. **Metadata** - TSV file that contains all of the relevant metadata per broad sample
+
+The platemap and metadata files are broken down by treatment (e.g., compound, crispr, or orf).
+
+We downloaded metadata files from [the metadata folder in AWS](https://cellpainting-gallery.s3.amazonaws.com/index.html#cpg0000-jump-pilot/source_4/workspace/metadata/external_metadata/).
+We downloaded the barcode platemap and platemap files from [the platemaps folder in AWS](https://cellpainting-gallery.s3.amazonaws.com/index.html#cpg0000-jump-pilot/source_4/workspace/metadata/platemaps/2020_11_04_CPJUMP1/).
+
+Visualization of the platemaps are as follows:
+
+![compound_platemap](./ex_figures/JUMP_compound_platemap.png)
+> Platemap for compound treatments
```

Comment:
Consider changing the platemap legend title from `values` to `Key` or something similar. I thin the first letter would look better capitalized.

Reply:
@MattsonCam What do you mean in line 16? And what are values and key in this context?

Reply:
Oh, I was more so talking about the platemap figure. On that figure the Key is titled "value", but I think applying the above suggestion would make it look better

Reply:
Ohhhh I got you! I think that should be an easy fix!

Reply:
Thanks for clarifying!